src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -963,13 +1324,9 @@ private static void DeleteCountOfRevisions(DocumentsOperationContext context, Sl
                     return;
                 }
 
-                if (configuration.Disabled == false && configuration.PurgeOnDelete)
+                if (configuration.PurgeOnDelete)
```

Comment:
should we go through `DeleteOldRevisionsForPutOrDelete` ?

Reply:
Done